### ISSUE TITLE
Keyboard.enabled not true behaviour

### ISSUE THREAD

**Issue by haxxmaxx** [qlik-oss/sn-table#339]
When keyboard.enabled is falsey, we need to make sure we don't interfere with the focus handlig too much.
- There should always be a tabindex=0 on one and only one of the table cells
- all pagination stuff should be tabable
- selections should not be canceled when you tab in/out
- keyboard.focus/blur should never be called
- Nothing should happen if keyboard.active changes

This basically means we need a sanity check everywhere where we call blur/focus and keep tabindexes as static as possible
